src/components/UI/UserlistCard.tsx

```diff
@@ -30,18 +34,17 @@ const UserlistCard = ({
   return (
     <div className={css["user_list_card"]}>
       <CardTitle />
-      <UserOwner />
-
-      {dummyData.map((obj) => (
+      {dummyUserData.map((obj) => (
         <User
           findUser={foundUserHandler}
-          data={dummyData}
+          data={dummyUserData}
           hovered={hoverHandler}
           id={obj.id}
           key={obj.id}
           pic={obj.pic}
           name={obj.name}
           email={obj.email}
```

Comment:
Use spread syntax and remove `data={dummyUserData}` prop.
It is the same array you're mapping over to display user data. As per other comment, move hover handler to this component and then you can eliminate this prop.